plugins/poll/lib/poll.rb

```diff
@@ -171,7 +274,23 @@ def self.serialized_voters(poll, opts = {})
                 AND po.poll_id = #{poll.id}
           ) v
           WHERE row BETWEEN #{offset} AND #{offset + limit}
-      SQL
+          ORDER BY digest, rank
+        SQL
+      else
+        votes = DB.query <<~SQL
+          SELECT digest, user_id
+            FROM (
+              SELECT digest
+                    , user_id
+                    , ROW_NUMBER() OVER (PARTITION BY poll_option_id ORDER BY pv.created_at) AS row
+                FROM poll_votes pv
+                JOIN poll_options po ON pv.poll_option_id = po.id
+                WHERE pv.poll_id = #{poll.id}
+                  AND po.poll_id = #{poll.id}
+            ) v
+            WHERE row BETWEEN #{offset} AND #{offset + limit}
+        SQL
```

Comment:
You shouldn't interpolate directly.

`DB.query(<<~SQL, foo: 1, bar: 2)`

Reply:
this is done.